app/src/main/java/net/skyscanner/backpack/demo/compose/CardButtonsStory.kt

```diff
@@ -0,0 +1,148 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButton
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonSize
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonStyle
+import net.skyscanner.backpack.compose.cardbutton.BpkCardButtonType
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkDimension
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun CardButtonsStory(
+  size: BpkCardButtonSize,
+  modifier: Modifier = Modifier,
+) {
+  LazyColumn(
+    modifier
+      .fillMaxSize()
+      .padding(top = BpkDimension.Spacing.Md)
+  ) {
+    item {
+      Row(
```

Comment:
could you extract this into a separate function with the style + background colour as a param and reuse to avoid having the three buttons defined multiple times per style

Reply:
done